src/main.js

```diff
@@ -6,7 +8,20 @@ import "./registerServiceWorker";
 import vuetify from "./plugins/vuetify";
 import "./style.css";
 import firebase from "@/firebase";
-import "material-design-icons-iconfont/dist/material-design-icons.css";
+//import "material-design-icons-iconfont/dist/material-design-icons.css";
+
+
+import cssVars from "css-vars-ponyfill";
+//import "material-design-icons-iconfont/dist/material-design-icons.css";	//import "material-design-icons-iconfont/dist/material-design-icons.css";
+
+
```

Comment:
this is just me being obnoxious but probably make it one line in between on either side than two

Reply:
Cleaned up.